examples/messagecommands.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+
+import json
+
+from flask import Flask
+
+# This is just here for the sake of examples testing
+# to make sure that the imports work
+# (you don't actually need it in your code)
+sys.path.insert(1, ".")
+
+from flask_discord_interactions import DiscordInteractions  # noqa: E402
+from flask_discord_interactions.context import ApplicationCommandType
+
+
+app = Flask(__name__)
+discord = DiscordInteractions(app)
+
+# Find these in your Discord Developer Portal, store them as environment vars
+app.config["DISCORD_CLIENT_ID"] = os.environ["DISCORD_CLIENT_ID"]
+app.config["DISCORD_PUBLIC_KEY"] = os.environ["DISCORD_PUBLIC_KEY"]
+app.config["DISCORD_CLIENT_SECRET"] = os.environ["DISCORD_CLIENT_SECRET"]
+
+
+# Clear any existing global slash commands
+discord.update_slash_commands()
+
+
+# Simple command to change a message
+@discord.command(name="Make it bold", type=ApplicationCommandType.MESSAGE)
+def boldMessage(ctx, message):
+    return f"**{message.content}**"
```

Comment:
I think something like this is a much cleaner way to do things. Right now, I've only got the `content` field hooked up.